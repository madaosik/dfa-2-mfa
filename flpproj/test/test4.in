2,1,3
cab
1
3,1
1,a,3
2,c,3
2,a,2
1,b,2
