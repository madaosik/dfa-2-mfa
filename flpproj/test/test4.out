0,1,2
abc
0
0,2
0,a,2
0,b,1
0,c,2
1,a,1
1,b,2
1,c,2
2,a,2
2,b,2
2,c,2
